charts/jindofsx/templates/config/jindofs-client-conf.yaml

```diff
@@ -17,9 +17,9 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
   {{- if $isSingleMaster }}
-  STORAGE_NAMESPACE_RPC_ADDRESS: {{ template "jindofs.fullname" . }}-master-0.{{ .Values.runtimeIdentity.namespace }}.svc{{ .Values.clusterDomain }}:{{ .Values.master.ports.rpc }}
```

Comment:
Please update chart version and change log. Thanks.

Reply:
done